src/features/auth/components/login-form.tsx

```
@@ -0,0 +1,118 @@
+    resolver: zodResolver(loginFormSchema),
+    defaultValues: {
+      email: "",
+      password: "",
+    },
+  });
+
+  const onSubmit = async (data: z.infer<typeof loginFormSchema>) => {
+    setIsLoading(true);
+    const result = await signInAction(data);
+
+    if (result && result.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)